docs/CONTRIBUTING.md

```diff
@@ -35,6 +39,38 @@ similar to the
 
 You may use `Signed-off-by`, but we're not requiring it.
 
+**General Commit Message Guidelines**:
+
+1. Title
+    - Specify the context or category of the changes e.g. `lib` for library changes, `docs` for document changes, `bin/<command-name>` for command changes, etc.
+    - Begin the title with the first letter of the first word capitalized.
+    - Aim for less than 50 characters, otherwise 72 characters max.
+    - Do not end the title with a period.
+    - Use an [imperative tone](https://en.wikipedia.org/wiki/Imperative_mood).
+2. Body
+    - Separate the body with a blank line after the title.
+    - Begin a paragraph with the first letter of the first word capitalized.
+    - Each paragraph should be formatted within 72 characters.
+    - Content should be about what was changed and why this change was made.
+    - If your commit fixes an issue, the commit message should end with `Closes: #<number>`.
+    - Adding `Approved by: ` to end the commit message is optional.
```

Comment:
Minor: we can get rid of this line too; the `Approved by:` line is completely transparent to contributors.